DataConnectors/Citrix_ADC_WAF.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id": "Citrix",
```

Comment:
Please rename id to CitrixNetscalerADCWAF so that data connector id is unique. We have Citrix Analytics data connector as well.

Reply:
Renamed to CitrixWAF.